src/Views/BucketPanel/EntryDetail.tsx

```diff
@@ -123,7 +139,95 @@ export default function EntryDetail(props: Readonly<Props>) {
     }
   };
 
+  const handleUpload = async (values: any) => {
+    setIsUploadLoading(true);
+    try {
+      if (!uploadFile) {
+        throw new Error("Please select a file to upload");
+      }
+
+      // Check write permissions
+      if (!props.permissions?.write) {
```

Comment:
we should check it during upload. Let's disable the button if there are no write permissions.

Reply:
you sill check the permission